src/test/java/com/flagsmith/FlagsmithClientUnitTest.java

```diff
@@ -0,0 +1,139 @@
+package com.flagsmith;
+
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+import static org.testng.Assert.assertEquals;
+import static org.testng.Assert.assertFalse;
+import static org.testng.Assert.assertNull;
+import static org.testng.Assert.assertTrue;
+
+public class FlagsmithClientUnitTest {
+
+  private FlagsmithClient.Builder builder;
+
+  @BeforeMethod(groups = "integration-offline")
```

Comment:
Is the group here intentionally different to the group of the tests below?

Reply:
oops... fixed